ddp-workspace/projects/ddp-singular/src/app/constants/route.ts

```diff
@@ -6,7 +6,7 @@ export enum Route {
   About = 'about',
   Login = 'login',
   Error = 'error',
-  Survey = 'survey',
+  Survey = 'survey/:id',
```

Comment:
It's better to set this url to `activity/:id` since all reminder emails that contain links to associated activities have this in their `href` attribute: `{{ baseWebUrl }}/activity/:id`